understanding/20/captions-live.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Captions (Live)</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Live videos have captions</dd>
-		<dt>Author task</dt><dd>Provide synchronized text for audio content in real-time videos</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Live videos have captions.</dd>
+         <dt>What to do</dt><dd>Provide synchronized text for audio content in real-time videos.</dd>
+         <dt>Why it's important</dt><dd>People who are deaf or hard of hearing can understand real-time video content.</dd> 
```

Comment:
Similarly, in understanding/20/audio-only-live.html, understanding/20/captions-live.html and understanding/20/captions-prerecorded.html:

Consider extending why it's important not just to include individuals who are deaf or hard of hearing, but also to include "and individuals with a cognitive disability that impacts attention and working memory, so that they have a way to access what they may have missed."